mingyjongo.js

```diff
@@ -85,8 +94,8 @@ client.on('message', (message) => {
             todaysMod %= mods.length
             srcom.getUserName(mods[todaysMod])
             .then(function(username){
-              if(username === 'Hyperresonance'){username = 'Hyper';}
-              discord_user = client.users.find(r => r.username === username);
+              //if(username === 'Hyperresonance'){username = 'Hyper';}
+              discord_user = client.users.get(modsToMessage[username]);
```

Comment:
Lazy evaluation would allow precision/control of this mod finding method, while maintaining the flexibility/automatic-syncing to speedrun.com changes that the old mod finding method had.

```suggestion
              if(modsToMessage[username]){
                  discord_user = client.users.get(modsToMessage[username]);
              }
              else{
                  discord_user = client.users.find(r => r.username === username);
                  modsToMessage[username] = discord_user.id;
                  // ToDo : store modsToMessage in .json 
              }
              
```